backend/pkg/api/users.go

```diff
@@ -12,40 +12,36 @@ type User struct {
 	Email string `json:"email"`
 }
 
-func GetEmails(authConfig auth.AuthConfig, userIDs []string, datasourceID int) []string {
-
+func GetEmails(authConfig auth.AuthConfig, userIDs []string, datasourceID int) ([]string, error) {
 	url := "http://" + authConfig.Username + ":" + authConfig.Password + "@" + "localhost:3000/api/tsdb/query"
-
 	queryString := "("
 	i := 0
 	for i < len(userIDs)-1 {
 		queryString += "'" + userIDs[i] + "'" + ", "
 		i += 1
 	}
 	queryString += "'" + userIDs[i] + "')"
-
-	body, e := NewQueryRequest("SELECT * FROM \"user\" WHERE id IN "+queryString, "0", "0", datasourceID).ToRequestBody()
-
-	if e != nil {
-		log.DefaultLogger.Error(e.Error())
+	body, err := NewQueryRequest("SELECT * FROM \"user\" WHERE id IN "+queryString, "0", "0", datasourceID).ToRequestBody()
```

Comment:
I *think* this reduces down to raw sql injection? Maybe theres some sanitisation? If not, still not really an issue here, but maybe something to think about in terms of security?

Reply:
It's certainly not very secure - but not sure what you're suggesting sorry